src/Api/Spot/User.php

```diff
@@ -127,6 +127,18 @@ public function getCapitalWithdrawHistory(array $data=[]){
         $this->data=$data;
         return $this->exec();
     }
+    
+    /**
+     * 提币 (USER_DATA)
+     * POST /sapi/v1/capital/withdraw/apply (HMAC SHA256)
+     */
+    public function postCapitalWithdraw(array $data=[]) {
+        $this->type = 'POST';
+        $this->path='/sapi/v1/capital/withdraw/apply';
```

Comment:
Maybe you can solve this better than I was able to with the hard-coded url.

Reply:
Thank you